src/constants/appConstants.tsx

```diff
@@ -39,3 +39,14 @@ export const DURATION_LIST: { label: string; value: number }[] = [
   { label: "30 Mins", value: 30 },
   { label: "60 Mins", value: 60 },
 ];
+
```

Comment:
As project gets bigger with each MR, the constants file will less sense if simply export without providing a clarity on where it will be used.
So how about this instead.

```
export const SESSIONS = {
BAR_PERCENT : 0.7,
BAR_BORDER_RADIUS: 6
}
```

Reply:
Makes sense. Done